### ISSUE TITLE
Vitals not displaying correctly after 10 entries

### ISSUE THREAD

**Issue by juggernautsei** [openemr/openemr#4537]
### Describe the bug

The form input boxes collapse to a width that is too small to enter data into the fields.

### To Reproduce

![image](https://user-images.githubusercontent.com/834958/126353632-b826949a-48e9-44a3-a69e-c96cb2d37ee7.png)

1. Add a '....'

### Expected behavior
fields stay wide enough to enter new data. 

### Client configuration

- Browser: All
- OpenEMR version: 6.0.0(2)
- Operating system: Ubuntu

**Comment by juggernautsei** [comment]
This fixes the issue. 

      .container input {
          min-width: 120px;
      }

**Comment by mdsupport** [comment]
Feedback from nurses is they want to see max nbr of past vitals. From that perspective it is better to move add/change to a popup.

**Comment by adunsulag** [comment]
@mdsupport Wouldn't it make sense to show the last 3-5 vitals and then display below the entire form a table containing all of the past vitals?  Either that or let the table be horizontally scrollable.  

Trying to play with padding / margin will only get you so far as you hit a maximum no matter what.

**Comment by sjpadgett** [comment]
I fixed this awhile back in master.

**Comment by stephenwaite** [comment]
thought that sounded familiar, https://github.com/openemr/openemr/commit/86b0822637245a719483426714ad0121f639bf2c#diff-6278040c8b5dad0be6d93080b7064aa4f2eddae26deaa2f6f2bef1b480df22c4

**Comment by sjpadgett** [comment]
Thanks @stephenwaite I'll try to remember next time I do a commit to v6, to add. Unless you remember too.

**Comment by mdsupport** [comment]
> display below the entire form a table containing all of the past vitals

Agree. Current approach works for many practices where visits are either related to an episode or regularly spaced follow ups. But in-patient use or when a patient is seen very frequently makes the current horizontal / viewport constrained format difficult to use. Table display opens possibility of an enhanced filtering (e.g. see vitals when b/p was > 160) that will be very useful.

**Comment by mdsupport** [comment]
On a marginally relevant note: display of uuid is never of any value to users.

**Comment by adunsulag** [comment]
Bumping this issue as we still have this problem.

**Comment by sjpadgett** [comment]
It still worked for column 1 but since adding a second column(Abn), not so much.
I increased min width and added min-width for Abn column:
iPad in portrait

![image](https://user-images.githubusercontent.com/1859985/137966384-94323403-dac2-47d3-bf5a-ee22e1c4b161.png)
